src/schema/ghga.yaml

```diff
@@ -951,7 +951,7 @@ classes:
         description: >-
           One or more Workflow entities associated with this Analysis.
         range: workflow
-        required: true
+        required: false
```

Comment:
What about reference_chromosome (Line 935). It is required, but it was not filled in the JSON of Marc?

Reply:
I am not sure if we should change that. It was not provided but the implementation team has not changed it to optional for this release.